tests/v2/validation/provisioning/rke2/cni_providers_test.go

```diff
@@ -0,0 +1,174 @@
+package rke2
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
```

Comment:
Hey Tim why do a whole new test file? Just curious, wouldn't it make sense to make these updates to the already existing node provider test?

Reply:
Yea good point. I removed the file and put the test under the existing node provider test. Thanks for the catch!